cmake/modules/SwiftSupport.cmake

```diff
@@ -27,6 +27,8 @@ function(get_swift_host_arch result_var_name)
     set("${result_var_name}" "armv7" PARENT_SCOPE)
   elseif("${CMAKE_SYSTEM_PROCESSOR}" STREQUAL "armv7l")
     set("${result_var_name}" "armv7" PARENT_SCOPE)
+  elseif("${CMAKE_SYSTEM_PROCESSOR}" STREQUAL "amd64")
+    set("${result_var_name}" "amd64" PARENT_SCOPE)
   elseif("${CMAKE_SYSTEM_PROCESSOR}" STREQUAL "AMD64")
```

Comment:
I wonder if we should change this to `MATCHES` instead.

Reply:
Is it Windows that aliases `AMD64` to `x86_64`, or is it something else?

Reply:
Windows does that.  Linux uses x86_64.